### ISSUE TITLE
Popup is missing accessible heading levels

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-hint#14]
## Subject of the issue
 For accessibility, all titles should be identified as a heading and include an appropriate aria-level for the hierarchy of the title.

- There isn't a main title for the popup, only items have titles. There is [`id="notify-heading"`](https://github.com/cgkineo/adapt-hint/blob/480f8a797b3928e5eb0a9c1b7a7a4950ca6dd43d/templates/hintPopup.hbs#L3C8-L3C27) which provides an `aria-label` for the notify popup but no on screen title or main title `aria-level="1"` exist.
- Item title [.hint-popup__item-title](https://github.com/cgkineo/adapt-hint/blob/480f8a797b3928e5eb0a9c1b7a7a4950ca6dd43d/templates/hintPopup.hbs#L10), isn't distinguishable as a title.

## Expected behaviour
- For consistency with Notify popup, we should support both a [main on screen title](https://github.com/adaptlearning/adapt-contrib-core/blob/98af66d3fa44c7a2cd79fc228c2e9e978ae1758a/templates/notifyPopup.hbs#L17) and an [altTitle](https://github.com/adaptlearning/adapt-contrib-core/blob/98af66d3fa44c7a2cd79fc228c2e9e978ae1758a/templates/notifyPopup.hbs#L23).
- Item title should have a `role="heading"` and `aria-level="2"`.
